src/urls/package-url-handler.ts

```diff
@@ -0,0 +1,149 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {lookupDependencyMapping} from '../manifest-converter';
+
+import {ConvertedDocumentFilePath, ConvertedDocumentUrl, OriginalDocumentUrl, PackageType} from './types';
+import {UrlHandlerInterface} from './url-handler-interface';
+import {getRelativeUrl} from './util';
+
+const isInBowerComponentsRegex = /(\b|\/|\\)(bower_components)(\/|\\)/;
+const isInNodeModulesRegex = /(\b|\/|\\)(node_modules)(\/|\\)/;
```

Comment:
`\b` matches `.` and ` ` and `:` for example.  Not sure if this is what you are really after.  Maybe these should just be `/(^|\/|\\)(bower_components)(\/|\\)/` and `/(^|\/|\\)(node_modules)(\/|\\)/`?

Reply:
lets chat in person, want to make sure this supports both URL formats ('./' prefixed & not prefixed)